src/UI/Implementation/Component/Input/ViewControl/Group.php

```diff
@@ -98,4 +98,10 @@ protected function getDataFactory(): DataFactory
     {
         return $this->data_factory;
     }
+
+    /** ATTENTION: see trait Group::_isClientSideValueOk! */
```

Comment:
Could we make this an annotation?
```suggestion
    /** ATTENTION: @see GroupInternals::_isClientSideValueOk() */
```